client/homebrew/editor/editor.jsx

```diff
@@ -96,7 +96,7 @@ const Editor = createClass({
 	},
 
 	getCurrentPage : function(){
-		const lines = this.props.brew.text.split('\n').slice(0, this.cursorPosition.line + 1);
+		const lines = this.props.brew.text.split('\n').slice(0, this.refs.codeEditor.getCursorPosition().line + 1);
 		return _.reduce(lines, (r, line)=>{
 			if(line.indexOf('\\page') !== -1) r++;
```

Comment:
Make sure to account for Legacy and V3 line splitting, since they are slightly different.

```
if((this.props.renderer == 'legacy' && line.includes('\\page')) ||
   (this.props.renderer == 'V3'     && line.match(/^\\page$/)))
{
...
}
```

Reply:
This should now be included in the latest commit.